PyNvCodec/TC/inc/MemoryInterfaces.hpp

```diff
@@ -31,6 +33,9 @@ enum Pixel_Format {
   BGR = 6,
   YCBCR = 7,
   YUV444 = 8,
+  RGB_32F = 9,
+  RGB_32F_PLANAR = 10,
+  RGB_32F_PLANAR_CONTIGUOUS = 11,
```

Comment:
May I ask you, what does contiguous planar image means? I assume that `RGB32F` is floating-point 3WxH interleaved RGB and `RGB_32F_PLANAR` is floating-poit Wx3H planar RGB, but what's `RGB_32F_PLANAR_CONTIGUOUS`?